<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammeManngerClass</class>
 <widget class="QWidget" name="ProgrammeManngerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgrammeMannger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#E0E0E0;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
     <item>
      <widget class="QTableWidget" name="tableWidget_Pro"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Current Recipe:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_programmName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:red;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_CreatePro">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QPushButton:hover
{
	background-color:#000426;
}
QPushButton:pressed
{
	background-color:rgb(65,65,65);
}
</string>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_DeletePro">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QPushButton:hover
{
	background-color:#000426;
}
QPushButton:pressed
{
	background-color:rgb(65,65,65);
}
</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ChangePro">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QPushButton:hover
{
	background-color:#000426;
}
QPushButton:pressed
{
	background-color:rgb(65,65,65);
}
</string>
         </property>
         <property name="text">
          <string>Set as Current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_DefaultPro">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QPushButton:hover
{
	background-color:#000426;
}
QPushButton:pressed
{
	background-color:rgb(65,65,65);
}
</string>
         </property>
         <property name="text">
          <string>Set as Default</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_SavePro">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QPushButton:hover
{
	background-color:#000426;
}
QPushButton:pressed
{
	background-color:rgb(65,65,65);
}
</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
